src/rhsmlib/dbus/base_object.py

```diff
@@ -94,6 +95,17 @@ def build_uep(self, options, proxy_only=False, basic_auth_method=False):
         else:
             return cp_provider.get_no_auth_cp()
 
+
+class BaseObject(dbus.service.Object):
+    # Name of the DBus interface provided by this object
+    interface_name = constants.INTERFACE_BASE
+    default_dbus_path = constants.ROOT_DBUS_PATH
+
+    def __init__(self, conn=None, object_path: str = None, bus_name: "BusName" = None):
+        if object_path is None:
+            object_path = self.default_dbus_path
+        super().__init__(conn=conn, object_path=object_path, bus_name=bus_name)
+
```

Comment:
The question is: Why do we have the method `GetAll()` in this class. I suspect that it returns always empty dictionary. Could we just remove it?

```
 | Look
 V
```

Reply:
OK, Thanks